docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -2,396 +2,1398 @@
-    "- [**View detailed drift dashboards**](#view-dashboards)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
+   "id": "699c9e94",
    "metadata": {},
    "source": [
     "## MLRun installation and configuration\n",
     "\n",
     "Before running this notebook make sure `mlrun` is installed and that you have configured the access to the MLRun service. "
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# Install MLRun if not installed, run this only once (restart the notebook after the install !!!)\n",
-    "%pip install mlrun tqdm ipywidgets"
-   ]
-  },
   {
    "cell_type": "markdown",
+   "id": "2ccd4c68",
    "metadata": {},
    "source": [
-    "<a id=\"setup-project\"></a>\n",
     "## Set up the project\n",
     "\n",
     "First, import the dependencies and create an [MLRun project](https://docs.mlrun.org/en/latest/projects/project.html). This  contains all of the models, functions, datasets, etc.:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import os\n",
-    "\n",
-    "import mlrun\n",
-    "import pandas as pd"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
+   "id": "c548d350-83c5-4886-8f96-401c8c139b7a",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "> 2023-03-12 17:02:37,120 [info] loaded project tutorial from MLRun DB\n"
+      "> 2024-03-27 11:22:46,181 [info] Loading project from path: {'project_name': 'mm-app-project', 'path': './'}\n",
+      "> 2024-03-27 11:23:01,621 [info] Project loaded successfully: {'project_name': 'mm-app-project', 'path': './', 'stored_in_db': True}\n"
      ]
     }
    ],
    "source": [
-    "project = mlrun.get_or_create_project(name=\"tutorial\", context=\"./\", user_project=True)"
+    "%config Completer.use_jedi = False\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "from sklearn.datasets import load_iris\n",
+    "import mlrun\n",
+    "from mlrun import import_function, get_dataitem, get_or_create_project\n",
+    "import uuid\n",
+    "\n",
+    "project_name = \"mm-app-project\"\n",
+    "project = get_or_create_project(project_name, context=\"./\")"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "9c8944e0-565a-4bc2-9515-e9a8f9724fb8",
    "metadata": {},
    "source": [
-    "```{admonition} Note\n",
-    "This tutorial does not focus on training a model. Instead, it starts with a trained model and its corresponding training dataset.\n",
-    "```\n",
+    "## Enable model monitoring\n",
+    "\n",
+    "Model monitoring is enabled per project. {py:meth}`~mlrun.db.httpdb.HTTPRunDB.enable_model_monitoring` brings up the controller and schedules it according to the `base_period`, and deploys the writer.\n",
```

Comment:
again, this is referencing httpd and the code goes to project, so need to be clear what to use

Reply:
fixed